open-data/Lesson6-Sharing&Reuse.md

```diff
@@ -0,0 +1,311 @@
+
+Each research community, either implicitly or explicitly, has its own pipeline for cleaning data, for example how to deal with missing values, normalize data and  so on. Likewise it is important to apply community-based measures of quality. It is therefore advisable to consult with data managers, librarians and other community members for common practices see lesson 5.4 Help.
+
+Data cleaning is important to provide access to accurate and consistent data, this is often highlighted in the phrase Garbage in-Garbage out, meaning that flawed data (e.g. duplicated or missing information) will produce incorrect or misleading outputs (e.g. statistics). Data quality can be reduced in several ways, including misspellings or typos during data entry, missing values, mixed formats, replicated entries of the same real-world entity or other invalid data
+
+This process requires significant human intervention (including in some cases sectoral or community efforts), which may consume time and energy. Anecdotally, preparing data in this fashion can take up to 80% of the time involved in the analysis of data.
+
+(Hellerstein, J. M. (2008). Quantitative data cleaning for large databases. United Nations Economic Commission for Europe (UNECE), 25, 1-42. [https://dataresponsibly.github.io/courses/documents/Hellerstein_2008.pdf](https://dataresponsibly.github.io/courses/documents/Hellerstein_2008.pdf)), (Chu, X., Ilyas, I. F., Krishnan, S., & Wang, J. (2016, June). Data cleaning: Overview and emerging challenges. In _Proceedings of the 2016 international conference on management of data_ (pp. 2201-2206. [https://doi.org/10.1145/2882903.2912574](https://doi.org/10.1145/2882903.2912574)).
+
+**Guides:**
+
+* Harvard- Analysis Ready Datasets [https://datamanagement.hms.harvard.edu/analyze/analysis-ready-datasets](https://datamanagement.hms.harvard.edu/analyze/analysis-ready-datasets)
+* [https://en.wikipedia.org/wiki/Data_cleansing#Data_quality](https://en.wikipedia.org/wiki/Data_cleansing#Data_quality)
+* [https://www.kaggle.com/learn/data-cleaning](https://www.kaggle.com/learn/data-cleaning)
+* [https://towardsdatascience.com/the-ultimate-guide-to-data-cleaning-3969843991d4](https://towardsdatascience.com/the-ultimate-guide-to-data-cleaning-3969843991d4) _ _
+* Carpentries [https://datacarpentry.org/lessons/](https://datacarpentry.org/lessons/)
+
+
+**Tools:**
+
+* [https://openrefine.org/](https://openrefine.org/)
+* See more in tools module
+
+
+## Summary
+
+
+Over the present decade, there has been a surge of interest from both industry and academia in Open Data, not only for record keeping but also to support a variety of decision-making tasks. Creating new data is costly and sometimes redundant. Therefore there is an increased emphasis on reuse of Open Data.
+
+
+In summary, sharing open data requires;
+
+1. planning for open data including for anonymizing or securing sensitive data where necessary;
```

Comment:
```suggestion
1. Planning for open data;
```
We do not talk about anonymising data here so I don't think it should be in the summary?